### PR TITLE
Add photo gallery to web portfolio

### PR DESCRIPTION
**Pull request by rtnavarr**
Added photo gallery feature to portfolio, as well as navigation links to and from the homepage. Also updated style.css to reflect changes in font(was changed from default to Montserrat) and image arrangement in the photo gallery.

### CODE REVIEW

File: rachelnavarro/step/portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -4,37 +4,40 @@
...
-	      <li>Attempting to skateboard. (And eating the pavement every. single. time.) </li>
-	      <li>Watching Tarantino films. </li>
-	      <li>Trying my hand at baking. (Still searching for the perfect vegan chocolate chip cookie recipe!) </li>
-	      <li>Scrolling endlessly through Tik Tok. (lol) </li>
-      </ul>
-
+      <!-- INTRODUCTION SECTION-->
+      <h3>About</h3>
+      <div class = "about">
+      	<img src="https://storage.cloud.google.com/rachelnavarro-step-2020.appspot.com/me.jpg" height="50%" width="50%">
```
Comment (andrewnguyen) [inline rachelnavarro/step/portfolio/src/main/webapp/index.html]: Please keep indentation consistent file wide

File: rachelnavarro/step/portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -4,37 +4,40 @@
...
+	<p>Hi there, and welcome to my portfolio!</p>
+	<p>I'm Rachel(she/her/hers), a current STEP intern here at Google. I'm
```
Comment (andrewnguyen) [inline rachelnavarro/step/portfolio/src/main/webapp/index.html]: space between Rachel and (

File: rachelnavarro/step/portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -4,61 +4,61 @@
...
+      <!-- INTRODUCTION SECTION-->
+      <h3>About</h3>
+      <div class = "about">
+        <img src="https://storage.cloud.google.com/rachelnavarro-step-2020.appspot.com/me.jpg" height="50%" width="50%">
+	<p>Hi there, and welcome to my portfolio!</p>
+	<p>I'm Rachel (she/her/hers), a current STEP intern here at Google. I'm 
+	   also a rising junior at UC Irvine(zot zot zot!), where I am majoring in Computer Science with a specialization 
+	   in Intelligent Systems. When I'm not coding, you can probably find me:
+        </p>
+      	<ul>
```
Comment (andrewnguyen) [inline rachelnavarro/step/portfolio/src/main/webapp/index.html]: indentation

Comment (rtnavarr) [reply rachelnavarro/step/portfolio/src/main/webapp/index.html]: Fixed the indentation on lines 18-30 @andrewnguyen @dchin2

File: rachelnavarro/step/portfolio/src/main/webapp/images.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        <img src="https://storage.cloud.google.com/rachelnavarro-step-2020.appspot.com/photo-gallery/gallery-9.jpg" width="100%" height="100%"/></a>
+        <img src="https://storage.cloud.google.com/rachelnavarro-step-2020.appspot.com/photo-gallery/gallery-11.jpg" width="80%" height="80%"/></a>
+        <img src="https://storage.cloud.google.com/rachelnavarro-step-2020.appspot.com/photo-gallery/gallery-14.jpg" width="50%" height="50%"/></a>
+        <img src="https://storage.cloud.google.com/rachelnavarro-step-2020.appspot.com/photo-gallery/gallery-15.jpg" width="50%" height="50%"/></a>
+        <img src="https://storage.cloud.google.com/rachelnavarro-step-2020.appspot.com/photo-gallery/gallery-16.jpg" width="100%" height="100%"/></a>
+        <img src="https://storage.cloud.google.com/rachelnavarro-step-2020.appspot.com/photo-gallery/gallery-17.jpg" width="50%" height="50%"/></a>
+        <img src="https://storage.cloud.google.com/rachelnavarro-step-2020.appspot.com/photo-gallery/gallery-18.jpg" width="80%" height="80%"/></a>
+      </div>
+      <p>Click <a href="index.html">here</a> to navigate back to the main page.</p>
```
Comment (dchin2) [inline rachelnavarro/step/portfolio/src/main/webapp/images.html:30]: Inconsistent indentation, please fix

File: rachelnavarro/step/portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -4,61 +4,60 @@
...
 	 to marginalized communities, and I can't wait to someday create tools that can be used to help drive positive societal change. 
       </p>
-
-      <!-- LINKS TO SOCIAL MEDIA- ORGANIZE INTO A TABLE -->
+      <!-- LINKS TO SOCIAL MEDIA -->
       <h3>Socials</h3>
       <p>Let's connect on social media!</p>
       <form action="https://www.linkedin.com/in/rachel-navarro-922b70191">
-	      <input type="submit" value="Add me on LinkedIn!" />
+        <input type="submit" value="Add me on LinkedIn!"/>
```
Comment (dchin2) [inline rachelnavarro/step/portfolio/src/main/webapp/index.html:39]: Indent two spaces here, instead of one.

**Review by dchin2 — CHANGES_REQUESTED** [review]
LGTM after indentation fixes

File: rachelnavarro/step/portfolio/src/main/webapp/index.html (hunk 5 above)
Comment (rtnavarr) [reply rachelnavarro/step/portfolio/src/main/webapp/index.html:39]: Just fixed the indentation! @dchin2

**Review by dchin2 — APPROVED** [review]
Nice

**Pull request merged**